examples/prepare_chx_shadow.py

```diff
@@ -0,0 +1,22 @@
+db.reg.register_handler("srw", SRWFileHandler, overwrite=True)
+db.reg.register_handler("shadow", ShadowFileHandler, overwrite=True)
+db.reg.register_handler("SIREPO_FLYER", SRWFileHandler, overwrite=True)
+
+plt.ion()
+
+root_dir = "/tmp/sirepo-bluesky-data"
+_ = make_dir_tree(datetime.datetime.now().year, base_path=root_dir)
+
+connection = SirepoBluesky("http://localhost:8000")
+
+data, schema = connection.auth("shadow", "I1Flcbdw")
```

Comment:
I don't think this simulation exists in the database for Sirepo started with the action from the sirepo-bluesky repo. We can add it if needed. Was it converted from the `HXV1JQ5c` SRW simulation (from [this list](https://nsls-ii.github.io/sirepo-bluesky/simulations.html#other-srw-simulations)) to the Shadow simulation?

Why not to use the existing TES Shadow simulation?